docs/src/main/paradox/amqp.md

```diff
@@ -115,6 +115,17 @@ We merge all sources into one and add the index of the source to all incoming me
 
 Such sink and source can be started the same way as in the previous example.
 
+### Using rabbitmq as an RPC mechanism
+
+If you have remote workers that you want to incorporate into a stream, you can do it using rabbit RPC workflow [RabbitMQ RPC](https://www.rabbitmq.com/tutorials/tutorial-six-python.html)
```

Comment:
Link to the Java one instead? ;) https://www.rabbitmq.com/tutorials/tutorial-six-java.html